### DISCUSSION TITLE
How did you find out about gdocwiki?

### DISCUSSION THREAD

**Discussion by gregwebs** [pingcap/gdocwiki#154] [General]
Also, what would be a good way for others to find this?

@gsipos

**Comment by gsipos** [comment]
Hi!
We were searching for a self-hosted/open-source alternative to https://youneedawiki.com/ as we didn't feel comfortable sharing our company sensitive documents with an another company. 

I think @bbazsy found it, so he could elaborate, but I think it was just searching on google

**Comment by gregwebs** [comment]
They have a feature request site for you need a wiki and I left a pointer to gdocwiki from there where gdocwiki had the requested features implemented. @bbazsy was that how you found it?
